junit5/src/main/java/cz/xtf/junit5/model/DockerImageMetadata.java

```diff
@@ -76,6 +77,12 @@ private static DockerImageMetadata getMetadata(OpenShift openShift, Image image)
             return metadataFromTag;
         }
 
+        //create namespace if not exists
+        if (openShift.getProject(openShift.getNamespace()) == null) {
+            openShift.createProjectRequest();
+            OpenShiftWaiters.get(openShift, () -> false).isProjectReady().waitFor();
```

Comment:
There should be used the same logic as in:
https://github.com/xtf-cz/xtf/blob/master/junit5/src/main/java/cz/xtf/junit5/listeners/ProjectCreator.java#L24-L56

so all secrets are setup. It would make sense to put above logic into separate method in `ProjectCreator` and delegate namespace creation to it.